<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3" rowspan="2">
     <widget class="QLineEdit" name="nameFileLe"/>
    </item>
    <item row="2" column="0" colspan="5">
     <widget class="QPushButton" name="loadCheckPb">
      <property name="text">
       <string>Загрузить файл для отображения формата json</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="nameFileLa">
      <property name="text">
       <string>Файл</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="5">
     <widget class="QTextEdit" name="resCheckTe"/>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLabel" name="infLa">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Выберите файл.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Дополнительно</string>
    </property>
    <addaction name="action_json"/>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_json">
   <property name="text">
    <string>Узнать про формат json</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Обрабатываемые ошибки синтаксиса</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
